webview-ui/src/index.css

```diff
@@ -45,3 +45,23 @@
   background-color: rgba(255, 255, 255, 0.5);
   color: brown;
 }
+
```

Comment:
Was tinkering with the layout, and if we do this:
```css
.pf-chatbot {
  height: 100vh;
}

.pf-chatbot__messagebox {
  width: 100%;
  max-width: 100% !important; /* to ignore media queries */
}

.pf-chatbot__jump {
  inset-inline-start: 95% !important;
}
```
then the chat view takes up more horizontal space with the scroll bar aligning with where the scroll bar is on the analysis view.


![image](https://github.com/user-attachments/assets/9a9c795b-ae07-42e1-b86a-178c8554c6de)